src/C-interface/ellblock/bml_import_ellblock_typed.c

```diff
@@ -84,7 +84,7 @@ bml_matrix_ellblock_t *TYPED_FUNC(
             }
             double norminf = TYPED_FUNC(bml_norm_inf)
                 (A_ij, A_bsize[ib], A_bsize[jb], A_bsize[jb]);
-            if (is_above_threshold(norminf, threshold))
+            if (is_norm_above_threshold(norminf, threshold))
```

Comment:
```suggestion
            if (norminf > threshold)
```

Reply:
Sorry, that comment somehow didn't make it. The 'dito' are all referencing this comment.

Reply:
Thanks @nicolasbock . Just to be clear, are you suggesting just doing x > y instead of going through the macro for these places? In other words, scrap the new macro and use the direct comparison?

Reply:
Yes that's what I was thinking.